script/generate-bindings

```diff
@@ -4,7 +4,6 @@ output_path=lib/binding_rust/bindings.rs
 header_path='lib/include/tree_sitter/api.h'
 
 bindgen                                        \
-  --size_t-is-usize                            \
```

Comment:
Just curious why you removed this one.

Reply:
This is because it's default behavior for the latest `bindgen-cli` and it doesn't have such CLI parameter anymore.
I also removed it on master and added protection to don't allow to use older version of the bindgen than the last used one.